### PR TITLE
Feat/search pane vertical content shift #5724

### PR DESCRIPTION
**Pull request by creido-welly**

### CODE REVIEW

File: src/components/SearchPane/SearchPane.tsx [hunk 1]
```diff
@@ -9,10 +9,37 @@ import SearchContext from 'SearchPane/Context/SearchPaneContext';
 import SearchPaneForm from './Form/SearchPaneForm';
 import SearchPaneControls from './Controls/SearchPaneControls';
 
+const CSS_CLASSES = {
+  IS_SEARCH_ACTIVE: 'search-open'
```
Comment (chriddell) [inline src/components/SearchPane/SearchPane.tsx]: A bit pernickety from me - but what do you think about renaming the selector to something stateful - e.g. `has-search-open`, `has-open-search-pane`, `has-open-search`?

Comment (creido-welly) [reply src/components/SearchPane/SearchPane.tsx]: I'll change to `is-search-active` (we're using `...active` for other selectors)

Comment (creido-welly) [reply src/components/SearchPane/SearchPane.tsx]: Done

**Review by chriddell — APPROVED** [review]

File: config/storybook/styles/_storybook-global.scss [hunk 2]
```diff
@@ -43,3 +43,17 @@ h5 {
...
+
+// stylelint-disable selector-max-id
+.is-search-active .header,
+.is-search-active .image-lead,
+.is-search-active .pride-strip,
+.is-search-active .banner,
+.is-search-active .main,
+.is-search-active #main,
+.is-search-active .footer {
+  // TODO: figure out why we can't use @include mq(md) here
```
Comment (chriddell) [inline config/storybook/styles/_storybook-global.scss]: Caused by the order of imports in storybook-app.scss.

The below will fix it:

```css
@import 'assets/styles/app';
@import './storybook-global';
```

**Review by chriddell — CHANGES_REQUESTED** [review]
See comment re: mq() in storybook.

File: config/storybook/styles/_storybook-global.scss (hunk 2 above)
Comment (creido-welly) [reply config/storybook/styles/_storybook-global.scss]: Sorted, worked fine with all of the extends ?!?!

Comment (chriddell) [reply config/storybook/styles/_storybook-global.scss]: Doesn't matter with extends it seems, they can be defined anywhere. I didn't know that, but good to find out!

**Review by chriddell — APPROVED** [review]

**Pull request merged**